cinema/views.py

```diff
@@ -20,25 +22,48 @@
     OrderListSerializer,
 )
 
+from user.permissions import IsAdminOrIfAuthenticatedReadOnly
+
 
-class GenreViewSet(viewsets.ModelViewSet):
+class GenreViewSet(
+    mixins.ListModelMixin, mixins.CreateModelMixin, viewsets.GenericViewSet
```

Comment:
Place each mixin and viewset on a new line for better readability here and in the other classes